src/pretalx/orga/forms/cfp.py

```diff
@@ -153,22 +156,45 @@ def __init__(self, *args, event=None, **kwargs):
         ):
             self.fields["is_public"].disabled = True
 
+    def clean(self):
+        deadline = self.cleaned_data["deadline"]
+        question_required = self.cleaned_data["question_required"]
+        if (deadline == "" or deadline is None) and (
```

Comment:
You can just use `if deadline` here.

Reply:
I made simpler the if conditions using ```not deadline``` and ```deadline```.